### ISSUE TITLE
Team marked an issue as duplicate but didn't respond to the original issue

### ISSUE THREAD

**Issue by QinHaichen12** [nus-cs2103-AY2425S1/forum#828]
![image](https://github.com/user-attachments/assets/ac8596bb-12b2-4749-812e-b2dabe9d05e1)
As per title, my issue was marked as a duplicate of another issue but the team didn't provide any response to the original issue. I am only left with the option to accept or reject the duplicate status. Does this mean the team has automatically accepted the original bug and how should I proceed in this case?

**Comment by damithc** [comment]
@QinHaichen12 Yes, the team has accepted the bug, type, and severity. You only need to decide if you agree with the duplicate flag.
